Messenger/AppDelegate.mm

```diff
@@ -755,6 +756,18 @@ - (WebView *)webView:(WebView *)sender createWebViewWithRequest:(NSURLRequest *)
 }
 
 
+- (NSArray *)webView:(WebView *)sender
+contextMenuItemsForElement:(NSDictionary *)element
+    defaultMenuItems:(NSArray *)defaultMenuItems {
+  #if DEBUG
+    if (([NSEvent modifierFlags] & NSControlKeyMask) == NSControlKeyMask) {
```

Comment:
FTR: Simply having the result of a mask operation be implicitly cast to a bool is usually more readable, i.e.
```mm
if ([NSEvent modifierFlags] & NSControlKeyMask) {
```